server/controllers/stock/inventory/receipts/list.handlebars

```diff
@@ -0,0 +1,65 @@
+<link href="{{absolutePath}}/css/bhima-bootstrap.css" rel="stylesheet">
+
+<div class="container-fluid">
+  <!-- header  -->
+  <div class="row">
+    <div class="col-xs-4">
+      <h3 style="margin:2px;">{{ model.enterprise.name }} ({{ model.enterprise.abbr}})</h3>
+      <span>{{translate 'FORM.LABELS.PHONE'}}: {{ model.enterprise.phone }}</span><br>
+      <span>{{translate 'FORM.LABELS.EMAIL'}}: {{ model.enterprise.email }}</span><br>
+    </div>
+    <div class="col-xs-4">
+      &nbsp;
+    </div>
+    <div class="col-xs-4 text-right">
+      <small>{{ model.project.name}} ({{ model.project.abbr }})</small>
+    </div>
+  </div>
+
+  <!-- body  -->
+  <div class="row">
+    <div class="col-xs-12">
+
+      <!-- page title  -->
+      <h1 style="margin:15px; font-weight:bold" class="text-center">{{uppercase (translate 'INVENTORY.LIST') }}</h1>
```

Comment:
If you only want uppercase, there is a simple style property that might be easier in the future:
```html
<h1 class="text-uppercase"></h1>
```
[Here is the bootstrap documentation](http://getbootstrap.com/css/#type-transformation).

Just for your information. :)